### PR TITLE
[Docs] [manage] yb-admin page updated with upgrade_ysql syntax

### PR DESCRIPTION
**Pull request by aishwarya24**
Summary:
- Added a note to **Upgrade a deployment** page for `upgrade_ysql` about Yugabyte version support (2.8 or higher).
- Updated **yb-admin** cli page with the details.

@netlify /latest/admin/yb-admin/

### CODE REVIEW

File: docs/content/latest/manage/upgrade-deployment.md [hunk 1]
```diff
@@ -110,7 +114,7 @@ For a successful YSQL upgrade, a message will be displayed as follows:
 YSQL successfully upgraded to the latest version
 ```
 
-In certain scenarios, YSQL upgrade can take longer than 60 seconds, which is the default timeout value for `yb-admin`. To account for that, run the command with a higher timeout value:
+In certain scenarios, a YSQL upgrade can take longer than 60 seconds, which is the default timeout value for `yb-admin`. To account for that, run the command with a higher timeout value:
```
Comment (polarweasel) [inline docs/content/latest/manage/upgrade-deployment.md:117]: What happens if yb-admin times out while the upgrade is running?

Comment (sushantrmishra) [reply docs/content/latest/manage/upgrade-deployment.md:117]: yeah, that is a good suggestions..

Comment (aishwarya24) [reply docs/content/latest/manage/upgrade-deployment.md:117]: Thank you @polarweasel and @sushantrmishra. Let me add it to the `yb-admin` page as well.

**Review by polarweasel — APPROVED** [review]
LGTM! Thank you @aishwarya24 and @sushantrmishra

**Review by sushantrmishra — APPROVED** [review]

**Pull request merged**